substrate/frame/identity/src/lib.rs

```diff
@@ -1005,8 +1005,9 @@ impl<T: Config> Pallet<T> {
 		Ok((new_id_deposit, new_subs_deposit))
 	}
 
-	/// Set an identity with zero deposit. Only used for benchmarking that involves `rejig_deposit`.
-	#[cfg(feature = "runtime-benchmarks")]
+	/// Set an identity with zero deposit. Used for benchmarking and XCM emulator tests that involve
+	/// `rejig_deposit`.
+	#[cfg(any(feature = "runtime-benchmarks", feature = "xcm-emulator-testing"))]
```

Comment:
As alternative the `runtime-benchmarks` feature can be enabled by default for this crate within the test crate, no additional feature will be needed